.gitignore

```diff
@@ -16,4 +16,4 @@ tmp.py
 .coverage
 .coverage.*
 htmlcov
-old_code/
\ No newline at end of file
+old_code/
```

Comment:
Acknowledged, I think this was just picked up because I was originally trying to exclude the *_pb2.py files from git history, but decided to include them so that external devs don't need to compile protos